src/xpk/core/core.py

```diff
@@ -2609,15 +2636,23 @@ def get_volume_mounts(args, system: SystemCharacteristics) -> str:
                 - name: shared-memory
                   mountPath: /dev/shm
                 - name: workload-terminated-volume
-                  mountPath: /usr/share/workload"""
+                  mountPath: /usr/share/workload
+                """
     elif system.device_type == h100_mega_device_type:
       volume_mount_yaml = """- name: nvidia-install-dir-host
                   mountPath: /usr/local/nvidia/lib64
                 - name: shared-memory
                   mountPath: /dev/shm
                 - name: workload-terminated-volume
-                  mountPath: /usr/share/workload"""
-
+                  mountPath: /usr/share/workload
```

Comment:
Can /usr/share/workload and other value sof mountPath in yamls be exported to variables?

Reply:
I like the idea, but I believe it's out of the scope of this PR